vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -343,6 +341,10 @@ class LEDGER_EXPORT Ledger {
 
   virtual void GetAllContributions(
       ledger::ContributionInfoListCallback callback) = 0;
+
+  virtual void SavePublisherInfo(
```

Comment:
Does iOS need to call this for any reason?

Reply:
nope, it's desktop specific usage for now